### DISCUSSION TITLE
Schema Dump includes setting migrations

### DISCUSSION THREAD

**Discussion by rabrowne85** [spatie/laravel-settings#292] [Q&A]
When you use the `php artisan schema:dump` command it builds the schema file, but because the settings migrations are included in the migrations table, these are included in the schema dump. As such any 'rebuild' from the schema file doesn't run the initial settings migrations.

As a work around I can 'remove' the lines from the schema file that inserts the settings migrations back in, but I'm wondering if there's another solution to this problem?

**Comment by rubenvanassche** [comment]
Normally we remove the settings when loading schema changes:

https://github.com/spatie/laravel-settings/blob/main/src/LaravelSettingsServiceProvider.php#L66-L113
